diagnostics/tropical_pacific_sea_level/settings.jsonc

```diff
@@ -0,0 +1,55 @@
+{
+  "settings" : {
+    "driver" : "tropical_pacific_sea_level.py",
+    "long_name" : "tropical pacific sea level",
+    "realm" : "ocns",
+    "description" : "Tropical pacific sea level diagnostic",
+    "runtime_requirements": {
+        "python3" : ["matplotlib", "xarray", "cartopy","cftime","numpy"]
+    },
+    "pod_env_vars" : {
+      "Model_name" : "CESM2_CORE",
+      "Model_legend_name" : "CESM2-CORE",
```

Comment:
The user-supplied name of the model being analyzed is set in the `CASENAME` environment variable; see [documentation](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_envvars.html#model-run-information)